how-to/what-is-vm-cache-and-how-do-I-use-it.md

```diff
@@ -0,0 +1,31 @@
+# What Is VMCache and How To Enable It
+
+### What is VMCache
+VMCache is a new function that creates VMs as caches before using it.
+It helps speed up new container creation.
+The function consists of a server and some clients communicating
+through Unix socket.  The protocol is GRPC in protocols/cache/cache.proto.
+The VMCache server will create some VMs and cache them by factory cache.
+It will convert the VM to GRPC format and transport it when gets
+requestion from clients.
+Factory grpccache is the VMCache client.  It will request GRPC format
+VM and convert it back to a VM.  If VMCache function is enabled,
+kata-runtime will request VM from factory grpccache when it creates
+a new sandbox.
+
```

Comment:
I think a common question is going to be "how is this different to VM templating?" so I would add a section to explain that and reference https://github.com/kata-containers/documentation/blob/master/how-to/what-is-vm-templating-and-how-do-I-use-it.md.

(Also, does https://github.com/kata-containers/documentation/blob/master/how-to/what-is-vm-templating-and-how-do-I-use-it.md need to reference this document?)

Reply:
> I think a common question is going to be "how is this different to VM templating?" so I would add a section to explain that and reference https://github.com/kata-containers/documentation/blob/master/how-to/what-is-vm-templating-and-how-do-I-use-it.md.
> 
> (Also, does https://github.com/kata-containers/documentation/blob/master/how-to/what-is-vm-templating-and-how-do-I-use-it.md need to reference this document?)

Added.